pkgs/applications/version-management/got/default.nix

```diff
@@ -64,10 +67,8 @@ stdenv.mkDerivation (finalAttrs: {
       Got. It will always remain possible to work with both Got and Git
       on the same repository.
     '';
-    homepage = "https://gameoftrees.org";
-    changelog = "https://gameoftrees.org/releases/CHANGES";
-    license = licenses.isc;
-    platforms = platforms.linux ++ platforms.darwin;
-    maintainers = with maintainers; [ abbe afh ];
+    maintainers = with lib.maintainers; [ abbe afh ];
+    mainProgram = "got";
+    platforms = with lib.platforms; [ darwin linux ];
```

Comment:
platforms.darwin and platforms.linux are both lists, and meta.platforms expect a list of platforms, not a list of lists.

Reply:
Thanks for pointing this out, @AndersonTorres, I'll read up on `platforms` seems there's still something I miss in my understanding of it.